.github/workflows/pull_request.yml

```
@@ -0,0 +1,61 @@
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with: 
+          node-version: '22.15'
+
+      - name: Check installed versions
+        run: |
+          node -v
+
+      - name: Restore node_modules cache
+        id: cache
+        uses: actions/cache@v3
```

actionlint (1.7.9)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)